toolbox/swiftsim/particle_splitting/generate_splitting_information.py

```diff
@@ -130,12 +165,28 @@ def load_snapshot(file_path):
         particle_ids = file.read(f"PartType{particle_type}/ParticleIDs")
         progenitor_ids = file.read(f"PartType{particle_type}/ProgenitorParticleIDs")
 
-        # Append to the final list only those which have split before. Additionally,
-        # we ignore all particles that have split more than the valid count (64) within SWIFT.
-        # Their split trees are not trustworthy. NOTE: if SWIFT is modified to handle those 
-        # splits without corrupt split trees, we could load them.
-        has_split = (counts > 0) & (counts < 64)
-
+        # TODO: What do we want to do for overflows if there is no overflow_data?
```

Comment:
I would argue we go to the current default behavior, which is to warn the user that splits beyond 64 are not being accounted for (if such an example exists).

Reply:
If the user does not pass a path to `splits.txt` files then we default back to the previous behaviour